src/multupd.jl

```diff
@@ -7,40 +7,54 @@
 #
 
 mutable struct MultUpdate{T}
-    obj::Symbol         # objective :mse or :div
-    maxiter::Int        # maximum number of iterations
-    verbose::Bool       # whether to show procedural information
-    tol::T              # change tolerance upon convergence
-    lambda::T           # regularization coefficient
+    obj::Symbol                 # objective :mse or :div
+    maxiter::Int                # maximum number of iterations
+    verbose::Bool               # whether to show procedural information
+    tol::T                      # change tolerance upon convergence
+    lambda_w::T                 # L1 regularization coefficient for W
+    lambda_h::T                 # L1 regularization coefficient for H
 
     function MultUpdate{T}(;obj::Symbol=:mse,
                             maxiter::Integer=100,
                             verbose::Bool=false,
                             tol::Real=cbrt(eps(T)),
-                            lambda::Real=sqrt(eps(T))) where T
+                            lambda_w::Real=zero(T),
+                            lambda_h::Real=zero(T),
+                            lambda::Union{Real, Nothing}=nothing) where T
 
         obj == :mse || obj == :div || throw(ArgumentError("Invalid value for obj."))
         maxiter > 1 || throw(ArgumentError("maxiter must be greater than 1."))
         tol > 0 || throw(ArgumentError("tol must be positive."))
-        lambda >= 0 || throw(ArgumentError("lambda must be non-negative."))
-
-        new{T}(obj, maxiter, verbose, tol, lambda)
+        lambda_w >= 0 || throw(ArgumentError("lambda_w must be non-negative."))
+        lambda_h >= 0 || throw(ArgumentError("lambda_h must be non-negative."))
+        if !isnothing(lambda) && lambda >= 0
```

Comment:
```suggestion
        if lambda !== nothing && lambda >= 0
```
`isnothing` was added in Julia 1.1 but this package still supports earlier versions. This is why CI is failing.

Reply:
Thank you. I understood it.